test/UriTest.php

```diff
@@ -222,10 +222,113 @@ public function testUri($rawUri, $expectedVals) {
         $this->assertEquals($expectedVals['fragment'], $uri->getFragment());
     }
 
+    public function testNormalize() {
+        $uri = new Uri('');
+        $this->assertEquals('', $uri->normalize());
+        $normalUri = 'http://google.com.tw/';
+        $uri = new Uri($normalUri);
+        $this->assertEquals($normalUri, $uri->normalize());
+    }
+
     public function testQueryParams() {
         $uri = new Uri('http://localhost/test.php?params=1&params=2');
         $expected = ['params' => [1, 2]];
         $actual = $uri->getAllQueryParameters();
         $this->assertEquals($expected, $actual);
     }
+
+    public function testGetAuthority() {
+        $uri = new Uri('http://www.google.com.tw:54467');
+        $expected = $uri->getAuthority();
+        $this->assertEquals($expected, 'www.google.com.tw:54467');
+    }
+
+    public function testGetAuthorityWithPassword() {
+        $validUri = 'ssh://username:password@example.com:123';
+        $uri = new Uri($validUri);
+        $password = 'password';
+        $this->assertEquals($password, $uri->getPass());
```

Comment:
You can just use `$this->assertEquals('password', $uri->getPass());` here, no need for an extra variable.